pkg/util/bpfutil/bpfexamples.go

```diff
@@ -0,0 +1,1593 @@
+// +build linux
+
+/*
+Copyright © 2019 LoT Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
Might want to add something like this, some parts/examples from this file are taken from https://github.com/iovisor/bpftrace/tree/master/tools which is also licensed under `Apache License, Version 2.0`

Reply:
Yes, this makes sense. I will add it.